packages/providers/mgt-msal-provider/src/MsalProvider.ts

```diff
@@ -208,7 +216,8 @@ export class MsalProvider extends IProvider {
     const scopes = options ? options.scopes || this.scopes : this.scopes;
     const accessTokenRequest: AuthenticationParameters = {
       loginHint: this._loginHint,
-      scopes
+      scopes,
+      extraQueryParameters: { domain_hint: this._domainHint }
```

Comment:
this should be added only if `this._domainHint` is set

Reply:
Updated by setting after:
b91ffd1